circle.yml

```diff
@@ -0,0 +1,7 @@
+machine:
+  python:
+    version: 3.4.3
+
+test:
+  override:
+    - ./runtests.sh
```

Comment:
[nits] [IMO] We don't have much tests to run, thus `python setup.py test` would be more concise.